vendor/assets/javascripts/google_analytics/analytics.js

```diff
@@ -0,0 +1,2412 @@
+(function() {
```

Comment:
Could we have minified version back again? It makes it easier to confirm that it's changed. Prettier should be ignoring `vendor/`